### ISSUE TITLE
numpy Installation missing libatlas-base-dev

### ISSUE THREAD

**Issue by bripley6** [piwheels/piwheels#329]
I recently installed numpy on RPI Zero running Python3.9. Upon trying to run #import numpy I get the error that libcblas.so.3 is missing:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/numpy/core/__init__.py", line 23, in <module>
    from . import multiarray
  File "/usr/local/lib/python3.9/dist-packages/numpy/core/multiarray.py", line 10, in <module>
    from . import overrides
  File "/usr/local/lib/python3.9/dist-packages/numpy/core/overrides.py", line 6, in <module>
    from numpy.core._multiarray_umath import (
ImportError: libcblas.so.3: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pi/code/plotly/plot.py", line 2, in <module>
    import numpy as np
  File "/usr/local/lib/python3.9/dist-packages/numpy/__init__.py", line 140, in <module>
    from . import core
  File "/usr/local/lib/python3.9/dist-packages/numpy/core/__init__.py", line 49, in <module>
    raise ImportError(msg)
ImportError: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy C-extensions failed. This error can happen for
many reasons, often due to issues with your setup or how NumPy was
installed.

We have compiled some common reasons and troubleshooting tips at:

    https://numpy.org/devdocs/user/troubleshooting-importerror.html

Please note and check the following:

  * The Python version is: Python3.9 from "/usr/bin/python3"
  * The NumPy version is: "1.23.4"

and make sure that they are the versions you expect.
Please carefully study the documentation linked above for further help.

Original error was: libcblas.so.3: cannot open shared object file: No such file or directory
```

It seems a required package for numpy doesn't get installed by default. I was able to get my import command to succeed by installing this package: `sudo apt-get install libatlas-base-dev`. Shouldn't this dependency automatically get installed or at least flagged from a `python3 -m pip install numpy` command?

FYI I installed numpy with this command:
```
sudo python3 -m pip install --force-reinstall numpy
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting numpy
  Using cached https://www.piwheels.org/simple/numpy/numpy-1.23.4-cp39-cp39-linux_armv6l.whl (12.4 MB)
Installing collected packages: numpy
  Attempting uninstall: numpy
    Found existing installation: numpy 1.23.4
    Uninstalling numpy-1.23.4:
      Successfully uninstalled numpy-1.23.4
Successfully installed numpy-1.23.4
```

FYI I was redirected here after reporting this issue in https://github.com/numpy/numpy/issues/22584

**Comment by bennuttall** [comment]
This is not the place to open a package issue. You have neglected to read the warning about this when opening a new issue here: https://github.com/piwheels/piwheels/issues/new/choose

Please refer to the numpy project page on piwheels.org: https://www.piwheels.org/project/numpy/

This page includes the necessary installation step

**Comment by bripley6** [comment]
Ah, thanks for that pointer and sorry for taking up your time on this!

Hopefully this ends up searchable for others - so I'll document some background. I was attempting to run plotly on a Raspberry Pi Zero W. Plotlyl's installation instructions are not Rpi specific and didn't call out all the dependencies in the quick start. Upon trying to run plotly I discovered I also needed panda. After getting panda installed I discovered numpy had issues, and I missed the [proper numpy install instructions](https://www.piwheels.org/project/numpy/) which also require running `sudo apt install libatlas3-base libgfortran5`. Hopefully others will find this information useful on resolving missing`libcblas.so.3` issue.
